server/protocols/webadmin/webadmin-swagger/src/main/java/org/apache/james/webadmin/swagger/RouteBuilder.java

```diff
@@ -0,0 +1,82 @@
+/****************************************************************
```

Comment:
We found with quynh while reading and exchanging about the code that this class was not needed and does nothing. It removes the **Route** limitation and allows to annotate whatever we want :-)